datavault-common/src/main/java/org/datavaultplatform/common/model/RoleAssignment.java

```diff
@@ -97,4 +97,14 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(id, role, user, school, vault);
     }
+
+    public String getTargetId() {
```

Comment:
Is anything using this?

Reply:
Yep, it's used in `RoleAssignmentController` so it doesn't have to do the `getGroup()`/`getVault()` check itself.